File: src/Services/MealService/Domain/Meal.Domain/Entities/Food.cs
using Meal.Domain.Common;
using Meal.Domain.Enums;

namespace Meal.Domain.Entities
{
    public class Food : BaseEntity
    {
        public Food()
        {
            Meals = new List<Meal>();
        }
        public required string Name { get; set; }
        public int CalorieByServing { get; set; }
        public FoodCategory Category { get; set; }

        public ICollection<Meal> Meals { get; set; }
    }
}

File: src/WebApps/Angular/AdminClient/src/app/contracts/meal/food/food.ts
import { Meal } from "../meal/meal";

export class Food {
    id: String;
    userId: String;
    status: number;
    createdDate: Date;
    updatedDate: Date | null | undefined;
    deletedDate: Date | null | undefined;
    name: String;
    calorieByServing: number;
    category: number;
    meals: Meal[];
}